config/authors/qamar_safadi.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Qamar A. Safadi",
+  "bio": "",
```

Comment:
As an Android Developer, I possess strong knowledge of Java, but I consider Kotlin to be my primary programming language.

Reply:
Updated